provider/local/release.go

```diff
@@ -172,6 +173,9 @@ func (p *Provider) ReleasePromote(app, id string) error {
 		return log.Error(err)
 	}
 
+	// clear current release cache so its no longer "active"
+	cache.Clear("storage", fmt.Sprintf("apps/%s/releases/%s/release.json", app, a.Release))
```

Comment:
I think its correct. The local release list was getting cached values of release.json, and that caching uses the "storage" key

https://github.com/convox/praxis/blob/master/provider/local/storage.go#L118